web2/htdocs/js/data.js

```diff
@@ -251,6 +251,7 @@
           var archive = this.data.archive[uuid];
           if (archive.tenant_uuid != q.tenant
            || ('system'  in q && archive.target_uuid  != q.system)
+           || ('job'     in q && archive.job          != q.job)
```

Comment:
I believe this is no longer needed, since we don't look up archives by job any more?  (that, and it didn't work too well to begin with?)